c8/client.py

```diff
@@ -4721,3 +4721,76 @@ def redis_sunionstore(self, destination, keys, collection):
             *keys,
         )
 
+    def list_billing_plans(self):
```

Comment:
I think the plan is to do client.plan().list_billing_plans()... etc basically just having a one method plan which provides the interface to connect to all the plan APIs... 
@dlozina-macrometa do correct me if I am wrong